globals.go

```diff
@@ -1072,13 +1072,20 @@ var SuperuserPermissions = []string{
 }
 
 const (
-	BasicProjectAccessRole = "basic_project_access"
-	BasicDistroAccessRole  = "basic_distro_access"
+	BasicProjectAccessRole     = "basic_project_access"
+	BasicDistroAccessRole      = "basic_distro_access"
+	SuperUserRole              = "superuser"
+	SuperUserProjectAccessRole = "admin_project_access"
```

Comment:
why is this ""admin_project_access" instead of "superuser_project_access"?

Reply:
That's just what it is in the backend already and it would require a migration to change it. I wanted to make the variable name more clear since project_admins are something else.